replication/binlogsyncer.go

```diff
@@ -28,6 +30,9 @@ type BinlogSyncerConfig struct {
 	// Flavor is "mysql" or "mariadb", if not set, use "mysql" default.
 	Flavor string
 
+	// MaxWaitTimeBetween Check connection
+	WaitTimeBetweenConnectionSeconds time.Duration
```

Comment:
```suggestion
	// WaitTimeBetweenConnectionSeconds is the delay between two connections when an connection fails
	WaitTimeBetweenConnectionSeconds time.Duration
```

Also, it's a time.Duration, so for naming you should avoid mentioning seconds


```suggestion
	// DelayBetweenConnection is the delay between two connections when an connection fails
	DelayBetweenConnection time.Duration
```